docs/components/modeler/desktop-modeler/element-templates/using-camunda-connector-templates.md

```diff
@@ -0,0 +1,28 @@
+---
+id: using-camunda-connector-templates
+title: Using Camunda Connector templates
+description: "Learn how to use Camunda Connector templates in Desktop Modeler."
+---
+
+Users of Camunda 8 and Desktop Modeler can use the same [Connector templates](../../../connectors/out-of-the-box-connectors/available-connectors-overview.md) that are available in Web Modeler.
+
+## Enable Camunda 8 Connector templates
+
+Camunda 8 Connector templates can be enabled through the `enable-connector-templates` [flag](../flags/flags.md). When enabled the templates will be kept up to date and you'll be able to use them in the Camunda 8 BPMN editor.
+
+On startup a notification will indicate whether the templates are up to date or have been updated.
+
+![Camunda Connector templates up to date notification](./img/using-camunda-connector-templates/up-to-date.png)
+
+In case of an error you'll see a nofication, too.
+
+![Error updating Camunda Connector templates notification](./img/using-camunda-connector-templates/error.png)
+
+With the templates enabled, you'll be able to use them in the Camunda 8 BPMN editor.
+
+![Using Camunda Connector templates in the Camunda 8 BPMN editor](./img/using-camunda-connector-templates/apply.png)
+
+## Additional resources
```

Comment:
I propose that we add an additional section to allow people to learn that they can "configure / manually download" connector templates of their choice using standard discovery mechanism (link to `configure-element-templates`).